main.tf

```diff
@@ -121,7 +124,7 @@ resource "ibm_database" "etcd_db" {
 
   ## This block is for if host_flavor IS NOT set
   dynamic "group" {
-    for_each = local.host_flavor_set ? [] : [1]
+    for_each = local.host_flavor_set && var.backup_crn == null ? [] : [1]
```

Comment:
That logic was wrong and later improved in postgresql to:

```icl
for_each = local.host_flavor_set == false && !local.recovery_mode ? [1] : []
```

Without pitr I think it needs to be 

```hcl
for_each = local.host_flavor_set == false && !var.backup_crn ? [1] : []
```